### PR TITLE
[T4A4][W09-A1] Low Yong Siang

### PR DESCRIPTION
**Pull request by YongSiang94**
Created a public interface Printable.

For Name, Email, Address and Phone classes, overridden the Printable “getPrintableString” method in the Printable class. 
And created a String for header formatting to use in getPrintableString method.

### CODE REVIEW

**Comment by YongSiang94** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -28,6 +29,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         }
         this.value = trimmedAddress;
     }
+    
+    @Override
+    public String getPrintableString(Printable... printables) {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Address.java:34]: No need to keep mentioning `Printable... printables`, recollect from tutorial discussion.

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Printable.java:3]: No header comment.

**Comment by jeffryhartanto** [comment]
@YongSiang94 
Some comments added. Please close the PR after reading comments. You forgot to combine the string, refer to the instruction.

**Pull request closed without merging**